test/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@
 add_executable(testregex testregex.cpp)
 add_test(testregexcorrect testregex)
 
-
 # correctness testing of checks
```

Comment:
In the command that you give, is the filename `${CMAKE_SOURCE_DIR}/test/aliceO2-namespace-naming.cpp` supposed to be an argument to the script? If so, then I should add the following line
`add_test(aliceo2namespace COMMAND check_clang_tidy.py ${CMAKE_SOURCE_DIR}/test/aliceO2-namespace-naming.cpp 'aliceO2-namespace-naming')`
Since the script accepts two arguments: 1st the file name, 2nd the name of the check. Right?